### DISCUSSION TITLE
No way to change label

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#127670] [Discussions] [answered]
For instance if I mislabeled a conversation as a question, there's no way to come back and change it to a different type later on. You're stuck with it!

**Comment by LiteBrite82** [comment]
You dont have this?
<img width="554" alt="Screenshot 2024-06-14 at 9 45 45 AM" src="https://github.com/community/community/assets/163985914/f6aca3d8-e21b-44f5-a8d3-ac3f2000596f">

**Reply by jidanni** [reply]
I have never seen that choice chart. 

Maybe it is because I'm not a supervisor. 

Maybe it is because I'm using the GitHub app. 

I just checked using this very conversation, and even the root article in this very conversation. I couldn't change anything.

**Reply by helloyanis** [reply, marked as answer]
I think it's because you don't have permissions to create labels on this specific conversation. Try it on your own repository by enabling discussions (it's in the settings tab, under General, there is a checkbox to check.)

Then you can create a new discussion topic and change labels by clicking on the ⚙️ at the side of the page, near "Labels".
![image](https://github.com/community/community/assets/40769776/181ed929-1534-40fc-9d99-9b9a604b4e44)
